internal/cmd/object-storage/credentials/create/create.go

```diff
@@ -0,0 +1,128 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	objectStorageUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/object-storage/utils"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/confirm"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/object-storage/client"
+	"github.com/stackitcloud/stackit-sdk-go/services/objectstorage"
+)
+
+const (
+	expiresFlag          = "expires"
+	credentialsGroupFlag = "credentials-group"
+	expirationTimeFormat = time.RFC3339
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	Expires            *time.Time
+	CredentialsGroupId string
+	HidePassword       bool
+}
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Creates credentials for a credentials group",
+		Long:  "Creates credentials for a credentials group.",
```

Comment:
It would be important to add an extra sentence saying that the credentials are only visible in creation. Something like:
`The credentials are only displayed upon creation, and it will not be recoverable later.`